pkg/storage/sqlstorage/migrations_test.go

```diff
@@ -155,8 +155,7 @@ var postMigrate = map[string]func(t *testing.T, store *sqlstorage.Store){
 			return
 		}
 	},
-	"1": func(t *testing.T, store *sqlstorage.Store) {},
-	"8": func(t *testing.T, store *sqlstorage.Store) {
```

Comment:
Why removing previous migration tests?

Reply:
The "1" is empty, so useless, and "8" was moved to "10" as we just need to check the state after all migrations are done.
This test is bloated (TestAllMigrations), i think we should remove it and prefer dedicated for each migration.

Reply:
Ok let's remove it then :D